### ISSUE TITLE
loss is negative and learning rate remain at 0

### ISSUE THREAD

**Issue by better-chao** [hologerry/SoCo#28]
During my model training, the training log shows that the loss values are negative, and the learning rate remains at 0. But the loss is still decreasing. Is this normal?
![image](https://github.com/hologerry/SoCo/assets/119278929/78b164d9-a443-4c07-bc55-1125640bc917)
